include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Solver/IDESolver.h

```diff
@@ -671,172 +686,180 @@ class IDESolver
+      FlowFunctionPtrType FlowFunc =
+          CachedFlowEdgeFunctions.getNormalFlowFunction(n, Fn);
       INC_COUNTER("FF Queries", 1, PAMM_SEVERITY_LEVEL::Full);
-      const container_type res =
-          computeNormalFlowFunction(flowFunction, d1, d2);
+      const container_type Res = computeNormalFlowFunction(FlowFunc, d1, d2);
       ADD_TO_HISTOGRAM("Data-flow facts", res.size(), 1,
                        PAMM_SEVERITY_LEVEL::Full);
-      saveEdges(n, fn, d2, res, false);
-      for (d_t d3 : res) {
-        EdgeFunctionPtrType g =
-            cachedFlowEdgeFunctions.getNormalEdgeFunction(n, d2, fn, d3);
+      saveEdges(n, Fn, d2, Res, false);
+      for (d_t d3 : Res) {      // NOLINT - keep close to algorithm notion
+        EdgeFunctionPtrType g = // NOLINT - keep close to algorithm notion
+            CachedFlowEdgeFunctions.getNormalEdgeFunction(n, d2, Fn, d3);
         LOG_IF_ENABLE(BOOST_LOG_SEV(lg::get(), DEBUG)
                       << "Queried Normal Edge Function: " << g->str());
-        EdgeFunctionPtrType fprime = f->composeWith(g);
+        EdgeFunctionPtrType fprime = // NOLINT - keep close to algorithm notion
+            f->composeWith(g);
         if (SolverConfig.emitESG()) {
-          intermediateEdgeFunctions[std::make_tuple(n, d2, fn, d3)].push_back(
+          IntermediateEdgeFunctions[std::make_tuple(n, d2, Fn, d3)].push_back(
               g);
         }
         LOG_IF_ENABLE(BOOST_LOG_SEV(lg::get(), DEBUG)
                           << "Compose: " << g->str() << " * " << f->str()
                           << " = " << fprime->str();
                       BOOST_LOG_SEV(lg::get(), DEBUG) << ' ');
         INC_COUNTER("EF Queries", 1, PAMM_SEVERITY_LEVEL::Full);
-        propagate(d1, fn, d3, fprime, nullptr, false);
+        propagate(d1, Fn, d3, fprime, nullptr, false);
       }
     }
   }
 
-  void propagateValueAtStart(const std::pair<n_t, d_t> nAndD, n_t n) {
+  void propagateValueAtStart(const std::pair<n_t, d_t> NAndD, n_t Stmt) {
     PAMM_GET_INSTANCE;
-    d_t d = nAndD.second;
-    f_t p = ICF->getFunctionOf(n);
-    for (const n_t c : ICF->getCallsFromWithin(p)) {
-      auto lookupResults = jumpFn->forwardLookup(d, c);
-      if (!lookupResults) {
+    d_t Fact = NAndD.second;
+    f_t Func = ICF->getFunctionOf(Stmt);
+    for (const n_t CallSite : ICF->getCallsFromWithin(Func)) {
+      auto LookupResults = JumpFn->forwardLookup(Fact, CallSite);
+      if (!LookupResults) {
         continue;
       }
-      for (auto entry : lookupResults->get()) {
-        d_t dPrime = entry.first;
-        EdgeFunctionPtrType fPrime = entry.second;
-        n_t sP = n;
-        l_t value = val(sP, d);
+      for (const auto &Entry :
+           LookupResults->get()) {   // TODO/FIXME: does the & break the solver?
+        d_t dPrime = Entry.first;    // NOLINT - keep close to algorithm notion
+        EdgeFunctionPtrType fPrime = // NOLINT - keep close to algorithm notion
```

Comment:
Define exceptions for clang-tidy so we don't need NOLINT everywhere

Reply:
Will do.